### PR TITLE
OAI-29: Claim bundle evaluation websocket endpoint

### PR DESCRIPTION
**Pull request by dborowiecki**
Ticket: [OAI-29](https://openimis.atlassian.net/browse/OAI-29)

### CODE REVIEW

File: claim_ai/evaluation/bundle_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class ClaimBundleEvaluation:
+
+    @classmethod
+    def evaluate_bundle(cls, claim_bundle):
+        entries = claim_bundle['entry']
+        evaluation_result = []
+        for next_claim in entries:
+            evaluation_result.append(cls._evaluate(next_claim))
+
+        return cls._build_resposne_bundle(evaluation_result)
```
Comment (dragos-dobre) [inline claim_ai/evaluation/bundle_evaluation.py]: Please correct spelling.

File: claim_ai/test_response_bundle.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
+adjudication_bundle = {
+    'resourceType': 'AdjudicationBundle',
+    'entry': [
+        {
+                "resourceType": "ClaimResponse",
+                "created": "2020-11-15",
+                "id": "C670E61A-36F1-4F70-A5D2-6CE2C20457F6",
+                "item": [
+                    {
+                        "adjudication": [
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py:23]: If this is an example for testing, please look in the specification https://openimis.atlassian.net/wiki/spaces/OP/pages/1790345259/openIMIS-AI+-+6.+Specification#Build-FHIR-ClaimResponse. The ClaimResponse will have only one adjudication item with the description in the specs.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: claim_ai/evaluation/bundle_evaluation.py (hunk 1 above)
Comment (dborowiecki) [reply claim_ai/evaluation/bundle_evaluation.py]: Fixed

File: claim_ai/test_response_bundle.py (hunk 2 above)
Comment (dborowiecki) [reply claim_ai/test_response_bundle.py:23]: Payload changed to meet the specification

File: claim_ai/test_response_bundle.py [hunk 3]
```diff
@@ -38,61 +16,18 @@
...
-                                    ],
-                                    "text": "valuated"
-                                },
-                                "reason": {
-                                    "coding": [
-                                        {
-                                            "code": "0"
-                                        }
-                                    ]
+                                    "text": "accepted"
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py:43]: The code 1 is for 'rejected'.

File: claim_ai/test_response_bundle.py [hunk 4]
```diff
@@ -120,79 +55,18 @@
...
-                                    ],
-                                    "text": "valuated"
-                                },
-                                "reason": {
-                                    "coding": [
-                                        {
-                                            "code": "0"
-                                        }
-                                    ]
+                                    "text": "rejected"
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py:82]: The code 0 is for 'accepted'. This is based on the AI Model output.

File: claim_ai/test_response_bundle.py [hunk 5]
```diff
@@ -211,45 +85,6 @@
                         ]
                     }
                 ],
-                "outcome": "valuated",
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py]: Set as "complete".

File: claim_ai/test_response_bundle.py [hunk 6]
```diff
@@ -211,45 +85,6 @@
...
-                "patient": {
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py]: You can leave this.

File: claim_ai/test_response_bundle.py [hunk 7]
```diff
@@ -211,45 +85,6 @@
...
-                    },
-                    {
-                        "number": 2,
-                        "text": "P"
-                    }
-                ],
-                "requestor": {
-                    "reference": "HealthcareService/64D5D616-ED4C-4DAC-A87A-19A7CAF01259"
-                },
-                "status": "Not Selected",
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py]: Mandatory

File: claim_ai/test_response_bundle.py [hunk 8]
```diff
@@ -8,28 +8,6 @@
                 "item": [
```
Comment (dragos-dobre) [inline claim_ai/test_response_bundle.py:21]: I let some comments after this, but is easier to follow based on the updated ClaimsResponse structure described in the specification: https://openimis.atlassian.net/wiki/spaces/OP/pages/1790345259/openIMIS-AI+-+6.+Specification#Build-FHIR-ClaimResponse

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Comment by dborowiecki** [comment]
@dragos-dobre I updated response bundle to match the documentation, can you take a look?

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**